go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bep/debounce v1.2.1 // indirect
 	github.com/bkielbasa/cyclop v1.2.0 // indirect
-	github.com/blackjack/webcam v0.0.0-20220329180758-ba064708e165 // indirect
+	github.com/blackjack/webcam v0.0.0-20230411204030-32744c21431f // indirect
```

Comment:
[note] all of these were updated either by the `go get -u go.viam.com/rdk@HEAD` or the `go mod tidy`